_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -0,0 +1,303 @@
+<?php
+/**
+ * REST API endpoint for admin menus.
+ *
+ * @package Jetpack
+ * @since 9.1.0
+ */
+
+/**
+ * Class WPCOM_REST_API_V2_Endpoint_Admin_Menu
+ */
+class WPCOM_REST_API_V2_Endpoint_Admin_Menu extends WP_REST_Controller {
+
+	/**
+	 * Namespace prefix.
+	 *
+	 * @var string
+	 */
+	public $namespace = 'wpcom/v2';
+
+	/**
+	 * Endpoint base route.
+	 *
+	 * @var string
+	 */
+	public $rest_base = 'admin-menu';
+
+	/**
+	 * WPCOM_REST_API_V2_Endpoint_Admin_Menu constructor.
+	 */
+	public function __construct() {
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
```

Comment:
`'show_in_index'       => false,` is missing. Is it `false` by default?

Reply:
Yeah that disables the OPTIONS endpoint. Not sure if the endpoint needs to be hidden, given that the code is open source, but I'm open to either. @jeherve do you have a preference?

Reply:
I think it would be fine to leave it open, since it's not publicly queryable anyway.

Reply:
Right, it only returns what you'd be able to see in wp-admin anyway